iiab-install

```diff
@@ -150,7 +150,7 @@ echo -e "Running local playbooks....Stage 0 will now run....followed by Stages $
 
 export ANSIBLE_LOG_PATH="$CWD/iiab-install.log"
 
-ansible -m setup -i $INVENTORY localhost --connection=local -e ansible_python_interpreter=/usr/bin/python3 | grep python
-ansible -m setup -i $INVENTORY localhost --connection=local -e ansible_python_interpreter=/usr/bin/python3 >> /dev/null
-ansible-playbook -i $INVENTORY $PLAYBOOK ${ARGS} --connection=local -e ansible_python_interpreter=/usr/bin/python3
+ansible -m setup -i $INVENTORY localhost --connection=local | grep python
+ansible -m setup -i $INVENTORY localhost --connection=local >> /dev/null
+ansible-playbook -i $INVENTORY $PLAYBOOK ${ARGS} --connection=local
```

Comment:
Does ${ARGS} belong on lines 153 and 154 too, similar to line 155 and ./runrole ?

Reply:
It hasn't in the past.